### PR TITLE
G-Mode in Big Pink part 3

### PR DESCRIPTION
**Pull request by kjbranch**
Should only be missing the remote acquiring of items now.

### CODE REVIEW

File: region/brinstar/pink.json [hunk 1]
```diff
@@ -1642,13 +1765,155 @@
...
+                  "name": "G-mode Morph Overload PLMs with Power Bombs and Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [8],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "SpringBall",
+                    {"ammo": { "type": "PowerBomb", "count": 4}},
```
Comment (osse101) [inline region/brinstar/pink.json:1781]:
```suggestion
                    "h_canArtificialMorphSpringBallBombJump",
                    {"ammo": { "type": "PowerBomb", "count": 3}},
```

File: region/brinstar/pink.json [hunk 2]
```diff
@@ -1642,13 +1765,155 @@
...
+                  "name": "G-mode Morph with Power Bombs and Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [8],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "SpringBall",
+                    {"ammo": {"type": "PowerBomb", "count": 3}}
```
Comment (osse101) [inline region/brinstar/pink.json:1819]:
```suggestion
                    "h_canArtificialMorphSpringBallBombJump",
                    {"ammo": {"type": "PowerBomb", "count": 2}}
```

File: region/brinstar/pink.json [hunk 3]
```diff
@@ -1642,13 +1765,155 @@
...
+                  "name": "G-mode Morph with Power Bombs and Spring Ball, Break Power Bomb Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [8],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "SpringBall",
+                    {"ammo": {"type": "PowerBomb", "count": 4}}
```
Comment (osse101) [inline region/brinstar/pink.json:1841]:
```suggestion
                    "h_canArtificialMorphSpringBallBombJump",
                    {"ammo": {"type": "PowerBomb", "count": 3}}
```

File: region/brinstar/pink.json [hunk 4]
```diff
@@ -1687,12 +1952,108 @@
...
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "Morph",
+                      {"and": [
+                        "SpringBall",
+                        "HiJump"
+                      ]},
+                      {"and": [
+                        "SpringBall",
+                        {"ammo": {"type": "PowerBomb", "count": 2}}
```
Comment (osse101) [inline region/brinstar/pink.json:1977]:
```suggestion
                        "h_canArtificialMorphSpringBallBombJump",
                        {"ammo": {"type": "PowerBomb", "count": 1}}
```

File: region/brinstar/pink.json [hunk 5]
```diff
@@ -1687,12 +1952,108 @@
...
+                    {"comeInWithGMode": {
+                      "fromNodes": [8],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "Morph",
+                      {"and": [
+                        "SpringBall",
```
Comment (osse101) [inline region/brinstar/pink.json:1972]: Seems hard to get past the Reo at 6

File: region/brinstar/pink.json [hunk 6]
```diff
@@ -1687,12 +1952,108 @@
...
+                  "name": "G-mode with Crystal Flash Clip and Overloading PLMs",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [9],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }},
+                    "h_canCrystalFlash"
+                  ],
```
Comment (osse101) [inline region/brinstar/pink.json:2002]:
```suggestion
                    "h_canCrystalFlash",
                    "canMidAirMorph"
                  ],
```

File: region/brinstar/pink.json [hunk 7]
```diff
@@ -1687,12 +1952,108 @@
...
+                    }},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "Morph",
+                      {"and": [
+                        "SpringBall",
+                        {"or": [
+                          "HiJump",
+                          "Bombs",
+                          {"ammo": {"type": "PowerBomb", "count": 4}}
```
Comment (osse101) [inline region/brinstar/pink.json:2032]:
```suggestion
                          "HiJump",
                          {"and": [
                            "h_canArtificialMorphSpringBallBombJump",
                            {"or": [
                              "Bombs",
                              {"ammo": {"type": "PowerBomb", "count": 3}}
                            ]}
                          ]}
```

File: region/brinstar/pink.json [hunk 8]
```diff
@@ -2277,6 +2638,32 @@
                     "With IBJ alone, it is easiest to IBJ from the bug pipe below to the ceiling, luring the Sidehoppers to the left.",
                     "At the top, overload PLMs using the camera scroll block just in front of the block, then roll through it."
                   ]
+                },
+                {
+                  "name": "G-mode Morph Overload PLMs with Power Bombs and Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    "SpringBall",
+                    {"ammo": { "type": "PowerBomb", "count": 3}},
```
Comment (osse101) [inline region/brinstar/pink.json:2647]:
```suggestion
                    "h_canArtificialMorphSpringBallBombJump",
                    {"ammo": { "type": "PowerBomb", "count": 2}},
```

File: region/brinstar/pink.json (hunk 5 above)
Comment (kjbranch) [reply region/brinstar/pink.json:1972]: Coming in from 8, the two Reos below the hopper pit spawn far to the left, so they shouldnt be a problem here

**Comment by kjbranch** [comment]
I haven't been always including low-level tech. in g-mode strats, for instance careful jumps. Bomb boost spring ball jumps are obscure enough that I think you're right that it is a good idea to always include them. On these strats, I'll assume that if they have bombs they will know how to IBJ and take that path instead of using bombs for their bomb boost spring ball jumps though.

File: region/brinstar/pink.json (hunk 5 above)
Comment (osse101) [reply region/brinstar/pink.json:1972]: Jumping over the pipe looks to make the reo move left.  And that is easy enough to do with springball.

**Pull request merged**